File: projects/GPL-FH-CSharp/features/DFS/WorkSpace.cs
/*layer DFS;
  // *************************************************************************
  */ 
public class WorkSpace 
{ // supply template actions
    public virtual void init_vertex( Vertex v ) {}
    public virtual void preVisitAction( Vertex v ) {}
    public virtual void postVisitAction( Vertex v ) {}
    public virtual void nextRegionAction( Vertex v ) {}
    public virtual void checkNeighborAction( Vertex vsource, 
                       Vertex vtarGet ) {}
}

File: plugins/de.ovgu.featureide.examples/featureide_examples/GPL-FH-Java/features/BFS/GPL/WorkSpace.java
package GPL;

import java.util.LinkedList;

// *************************************************************************
   
public class WorkSpace 
{ // supply template actions
    public void init_vertex( Vertex v ) {}
    public void preVisitAction( Vertex v ) {}
    public void postVisitAction( Vertex v ) {}
    public void nextRegionAction( Vertex v ) {}
    public void checkNeighborAction( Vertex vsource, 
     Vertex vtarget ) {}
}
